### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by sprd0x**
please review

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -9,8 +9,14 @@ public static int ekub(int a, int b) {
     return 0;
     }
     public static int ekuk(int a, int b) {
-        //TODO Aziz
-        return 0;
+        int e = 0;
+        for (int i = 1; i <1000000; i++) {
+            if (i % a == 0 && i % b == 0) {
```
Comment (Sardorbek20032004) [inline src/Main.java:14]: ekuk bunaqa xisoblanmaydi

Comment (zilola13) [reply src/Main.java:14]: togri aziz aka xato yozgansz

Comment (sprd0x) [reply src/Main.java:14]: Hop

**Comment by Sardorbek20032004** [comment]
yaxshi

**Pull request merged**